modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -283,6 +287,143 @@ For production {product-title} clusters on which you want to perform installatio
 ====
 endif::private[]
 
+ifdef::restricted[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes <7>
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  ibmcloud:
+    region: us-east <1>
+    resourceGroupName: us-east-example-network-rg <8>
```

Comment:
I do not think this is a MUST, and `resourceGroupName` is used to specify where the cluster is deployed to, not have to share with VPC resources.

Reply:
```suggestion
    resourceGroupName: us-east-example-cluster-rg <8>
```